Pinta.Core/Classes/ToolOption/IntegerOption.cs

```diff
@@ -86,46 +83,27 @@ public string GetUniqueName ()
 		return name;
 	}
 
-	public object GetValue ()
-	{
-		return value;
-	}
-
 	public void SetValue (object value)
```

Comment:
Is it possible to change this to `SetValue(int value)` to be more strongly-typed? From a quick look I think it's now always being called with an `int` anyways

If so, this could probably just be moved into the `set` method for the `Value` property